storage/database/src/main/java/de/danoeh/antennapod/storage/database/PodDBAdapter.java

```diff
@@ -260,7 +265,9 @@ public class PodDBAdapter {
             + TABLE_NAME_FEED_ITEMS + "." + KEY_ITEM_IDENTIFIER + ", "
             + TABLE_NAME_FEED_ITEMS + "." + KEY_IMAGE_URL + ", "
             + TABLE_NAME_FEED_ITEMS + "." + KEY_AUTO_DOWNLOAD_ATTEMPTS + ", "
-            + TABLE_NAME_FEED_ITEMS + "." + KEY_PODCASTINDEX_CHAPTER_URL;
+            + TABLE_NAME_FEED_ITEMS + "." + KEY_PODCASTINDEX_CHAPTER_URL + ", "
+            + TABLE_NAME_FEED_ITEMS + "." + KEY_PODCASTINDEX_TRANSCRIPT_TYPE + ", "
+            + TABLE_NAME_FEED_ITEMS + "." + KEY_PODCASTINDEX_TRANSCRIPT_URL;
```

Comment:
I guess there's no technical impact, but if possible I think we should rename this from PodcastIndex to PodcastNamespace. While initiated by the same folks, the namespace data can continue to exist in feeds without the Index. (This ofc would then apply also to all other instances of `Index`.)

Reply:
How about shorter PodcastNs ?

Reply:
Yeap, I was also thinking about something like that but I wasn't sure if it'd be too abstract.

Would it make sense to do the refactoring for chapters in Dev branch and already include it in a regular release before transcrips?